AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginLazyLoadTests/LL1/GraphQLLazyLoadPostComment4V2Tests.swift

```diff
@@ -145,25 +145,21 @@ final class GraphQLLazyLoadPostComment4V2Tests: GraphQLLazyLoadBaseTest {
                   updatedAt
                   post {
                     id
+                    createdAt
```

Comment:
Add test document at the top, Given / When / Then format. Also remove "// This looks broken" comment